src/main/java/uk/gov/cshr/controller/emailUpdate/EmailUpdateController.java

```diff
@@ -0,0 +1,112 @@
+package uk.gov.cshr.controller.emailUpdate;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+import uk.gov.cshr.controller.form.EmailUpdatedRecentlyEnterTokenForm;
+import uk.gov.cshr.domain.Identity;
+import uk.gov.cshr.domain.OrganisationalUnitDto;
+import uk.gov.cshr.exception.ResourceNotFoundException;
+import uk.gov.cshr.repository.IdentityRepository;
+import uk.gov.cshr.service.CsrsService;
+import uk.gov.cshr.service.EmailUpdateService;
+
+import javax.validation.Valid;
+import java.util.Optional;
+
+@Slf4j
+@Controller
+@RequestMapping("/emailUpdated")
+public class EmailUpdateController {
+
+    private static final String EMAIL_UPDATED_RECENTLY_ENTER_TOKEN_FORM = "emailUpdatedRecentlyEnterTokenForm";
+
+    private static final String ENTER_TOKEN_SINCE_EMAIL_UPDATE_VIEW_NAME = "enterTokenSinceEmailUpdate";
```

Comment:
Worth explicitly identifying these as templates in the var name

Reply:
Updated